src/importBook.js

```diff
@@ -299,15 +300,21 @@ export class ImportBook {
                             if (isV10) {
                                 token.actorId = actor._id;
                                 if (
-                                    (!token.texture?.src || token.texture?.src === "icons/svg/mystery-man.svg") &&
+                                    (!token.texture?.src ||
+                                        token.texture?.src === "icons/svg/mystery-man.svg" ||
+                                        token.texture?.src?.includes("placeholder") ||
+                                        token.texture?.src?.includes("-token")) &&

```

Comment:
I'm not certain about the `.includes` here as opposed to `.endsWith`, but as long as the data adheres to this convention, it should be fine.